src/release/ext-iau.xml

```diff
@@ -0,0 +1,31 @@
+<assembly>
+  <id>iau-plugin</id>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <fileSets>
+    <fileSet>
+      <directory>release/extensions/iau</directory>
+      <outputDirectory></outputDirectory>
+      <includes>
+         <include>*</include>
+      </includes>
+    </fileSet>
+    <fileSet>
+       <directory>release/target/dependency</directory>
+       <outputDirectory></outputDirectory>
+       <includes>
+          <include>gt-iau-wkt*.jar</include>
+       </includes>
+    </fileSet>
+    <fileSet>
+      <directory>release/target/html/licenses</directory>
+      <outputDirectory></outputDirectory>
+      <includes>
+         <include>LGPL.html</include>
```

Comment:
Do the definitions have a data license at all?

Reply:
The jar contains also Java code, the SPI needs a dedicated subclass of the property based authority to work, even if the overrides are minimal (hence LGPL). I have no idea about the definitions.